src/status_im/ui/screens/profile/visibility_status/views.cljs

```diff
@@ -25,17 +25,11 @@
   (re-frame/dispatch
    [:visibility-status-updates/delayed-visibility-status-update status-type]))
 
-;; In new ui, we are allowing switcher to overlap status-bar (draw over status bar)
-;; that's why the measure will return height including, the height of the status bar in android
-;; for calculating the correct position of the button on the profile screen, we have to decrease this height
```

Comment:
Not required anymore, probably because status bar `drawBehind=true`. IOS and android are behaving similar

Reply:
is there any other parts of the codebase where had issues like this?

Reply:
unrelated to your p.r of course :)